Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -97,11 +98,113 @@ Detector::Detector()
 {
 }
 
-Detector::Detector(const char *name, Bool_t active, const Int_t nlay)
+static double radii2Turbo(double rMin, double rMid, double rMax, double sensW)
+{
+  // compute turbo angle from radii and sensor width
+  return TMath::ASin((rMax * rMax - rMin * rMin) / (2 * rMid * sensW)) * TMath::RadToDeg();
+}
+
+static void configITS(Detector *its) {
+  // build ITS upgrade detector
+  // pALPIDE3 15x30 mm^2  (X,Z) with 26.88 x 29.24 micron pitch
+  const double kSensThick = 18e-4;
+  const double kPitchZ = 29.24e-4;
+  const double kPitchX = 26.88e-4;
+  const int    kNRow   = 512; 
+  const int    kNCol   = 1024;
+  const double kSiThickIB = 50e-4;
+  const double kSiThickOB = 50e-4;
+  //  const double kSensThick = 120e-4;   // -> sensor Si thickness
+  //
+  const double kReadOutEdge = 0.12;   // width of the readout edge (passive bottom)
+  const double kTopEdge = 37.44e-4;   // dead area on top
+  const double kLeftRightEdge   = 29.12e-4; // width of passive area on left/right of the sensor
+  //
+  const int kNLr = 7;
+  const int kNLrInner = 3;
+  const int kBuildLevel = 0;
+  enum { kRmn, kRmd, kRmx, kNModPerStave, kPhi0, kNStave, kNPar };
+  // Radii are from last TDR (ALICE-TDR-017.pdf Tab. 1.1, rMid is mean value)
+  const double tdr5dat[kNLr][kNPar] = {
+    {2.24, 2.34, 2.67,  9., 16.42, 12}, // for each inner layer: rMin,rMid,rMax,NChip/Stave, phi0, nStaves
+    {3.01, 3.15, 3.46,  9., 12.18, 16},
+    {3.78, 3.93, 4.21,  9.,  9.55, 20},
+    {-1,  19.6 ,   -1,  4.,  0.  , 24},  // for others: -, rMid, -, NMod/HStave, phi0, nStaves // 24 was 49
+    {-1,  24.55, -1,    4.,  0.  , 30},  // 30 was 61
+    {-1,  34.39, -1,    7.,  0.  , 42},  // 42 was 88
+    {-1,  39.34, -1,    7.,  0.  , 48}   // 48 was 100
+  };
+  const int nChipsPerModule = 7; // For OB: how many chips in a row
+  const double zChipGap = 0.01;  // For OB: gap in Z between chips
+  const double zModuleGap = 0.01;// For OB: gap in Z between modules
+
+  // Delete the segmentations from previous runs
+  std::remove("itsSegmentations.root");
```

Comment:
For consistency it is better to use ````std::remove(o2::ITS::GeometryTGeo::getITSSegmentationFileName());````

Later I'll add a fix to avoid at all itsSegmentation.root in the default configuration, generating it internally unless it is overridden for some non-standard configuation
(it was introduced in the beginning of ITSup studies when we thought we may have different detector types: strips, pixels etc).